package.json

```diff
@@ -165,15 +165,45 @@
       "collapsed": true,
       "properties": {
         "enabled": {
-          "title": "Use asciidoctor-pdf",
-          "description": "[asciidoctor-pdf](https://github.com/asciidoctor/asciidoctor-pdf) must be installed in your system.",
+          "title": "Enable PDF export using Asciidoctor PDF.",
+          "description": "The [asciidoctor-pdf](https://github.com/asciidoctor/asciidoctor-pdf) command must be available on your `PATH`.",
```

Comment:
It seems it can go either way. But "in your PATH" seems to be used more frequently.